app/views/content/international-returners/_read-up-on-changes-to-english-education.md

```diff
@@ -5,5 +5,5 @@ Browse the latest guidance:
 * [Curriculum and qualifications](https://www.gov.uk/schools-colleges-childrens-services/curriculum-qualifications)
```

Comment:
should these bullet points start with lowercase?

Reply:
@laurahermionetennant92 Not everything, but we try to where we can. I think it makes sense to in this case as the other bullet points on this page all start lowercase so is good for consistency/readability

Reply:
@laurahermionetennant92 I agree with Ellie - for points of style, we should use GDS standards where possible even as a campaign site, both for consistency and because they have been tested for accessibility/readability

However, I'll publish the page as is now given that this PR has been outstanding for a while - something to add for the next update!